### ISSUE TITLE
Document every widget traitlet.

### ISSUE THREAD

**Issue by minrk** [jupyter-widgets/ipywidgets#644]
# Summary

Every widget trait declared in the [python files](https://github.com/jupyter-widgets/ipywidgets/tree/master/ipywidgets/widgets) (except perhaps the traits starting with an underscore) should have a `help` docstring, like [this](https://github.com/jupyter-widgets/ipywidgets/blob/883ff53a7cd7c9546b0ba0c79d6b8128a6423d15/ipywidgets/widgets/widget_button.py#L24):

```
    button_color = Color(None, allow_none=True, help="Color of the button").tag(sync=True)
```

---
# Original issue

We need a lot more docstrings in ipywidgets to improve the experience of learning widgets in IPython with `?`, etc.

**Comment by SylvainCorlay** [comment]
Agreed.

**Comment by jasongrout** [comment]
by the way, with the switch to typescript, we have api docs by doing `npm run docs`.

**Comment by akhmerov** [comment]
Could this be solved by ipython/traitlets#261?

**Comment by jasongrout** [comment]
Thanks for pointing that out!

Bumping to 6.1.
